portfolio/src/main/webapp/images.html

```diff
@@ -12,24 +12,23 @@
             <li><a href="projects.html">Projects</a></li>
             <li><a href="images.html" class="active">Image Gallery</a></li>
         </ul>
+
         <h1>Memory Palace Tour</h1>
         <div class="ImgGallery">
             <p>Welcome to my Memory Palace!</p>
             <p>This is me.</p>
             <img src="images/headphone.jpeg">
```

Comment:
Please add 'alt' for all <img> tag. https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback